bittorrent/bittorrent.go

```diff
@@ -110,12 +110,22 @@ type Scrape struct {
 	Incomplete uint32
 }
 
+// AddressFamily is the address family of an IP address.
+type AddressFamily uint8
+
+// AddressFamily constants.
+const (
+	IPv4 = AddressFamily(iota)
```

Comment:
That's interesting syntax.
I've always written enums `IPv4 AddressFamily = iota`
I'm not sure which is more idiomatic.